applications/app/controllers/OptInController.scala

```diff
@@ -29,16 +29,12 @@ class OptInController extends Controller {
 
   def handle(feature: String, choice: String) = Action { implicit request =>
     Cached(60)(WithoutRevalidationResult(feature match {
-      case "https" => https.opt(choice)
       case "hsts" => hsts.opt(choice)
```

Comment:
I don't believe that we need any of this any longer as everyone should always be on HTTPS.

HSTS is now also on generally, so I believe can also go, though @mchv might have an opinion.